content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Installing External Control Plane [experimental]
+description: Install external control plane and remote config cluster.
+weight: 46
+keywords: [external,control,istiod,remote,config]
+aliases:
+    - /docs/setup/kubernetes/additional-setup/external-controlplane/
+owner: istio/wg-environments-maintainers
+test: no
```

Comment:
Should be yes? You're planning on testing it, right? This just controls the auto-generation of the snips.sh file.

Reply:
Yes, planning to test in this PR - https://github.com/istio/istio.io/pull/8431